### ISSUE TITLE
[Guide] Missing dependencies on Debian bullseye

### ISSUE THREAD

**Issue by etherealxx** [ungoogled-software/ungoogled-chromium-debian#340]
Guide for Debian bullseye user:

- `libflac12` and `libjsoncpp25` isn't available on bullseye. Those packages (*.deb) are available to download on [packages.debian.org](https://www.debian.org/distrib/packages), download the bookworm version, install them with `dpkg`
- the older version of `ungoogled-chromium-libraries` and `ungoogled-chromium-sandbox` needs to be uninstalled first before installing the newer ungoogled-chromium package

just gonna put it here as a note

**Comment by iskunk** [comment]
> download the bookworm version

Generally, you won't be able to install bookworm packages on bullseye, since bookworm uses a newer glibc version.

Which ungoogled-chromium package is this in reference to? Is it the version 112 build for unstable/sid?

**Comment by etherealxx** [comment]
yes the one i got from [here](https://software.opensuse.org//download.html?project=home%3Aungoogled_chromium&package=ungoogled-chromium)
tbh i have little knowledge with linux stuff, i just put note on what i did and works, might useful for someone else, or maybe someone had better method/alternative.
feel free to maybe ask things about my device if you need to.
